.github/workflows/scripts/prepare-test-summary-daily.js

```diff
@@ -0,0 +1,154 @@
+/**
```

Comment:
This file is similar to `.github/workflows/scripts/prepare-test-summary.js` and serves the same purpose: to post the test summary of the daily smoke test results, like so:

<img width="746" alt="XkJ8QBAtsE" src="https://user-images.githubusercontent.com/4509348/196179204-4cba0a35-251a-4eaa-8564-4a16cecd7102.png">